tools/functional-tester/etcd-tester/tester.go

```diff
@@ -160,37 +162,37 @@ func (tt *tester) compact(rev int64, timeout time.Duration) (err error) {
 		}
 	}()
 
-	plog.Printf("%s compacting storage (current revision %d, compact revision %d)", tt.logPrefix(), tt.currentRevision, rev)
+	plog.Printf("%s compacting storage (current revision %d, compact revision %d)", tt.LogPrefix(), tt.currentRevision, rev)
 	if err = tt.cluster.compactKV(rev, timeout); err != nil {
 		return err
 	}
-	plog.Printf("%s compacted storage (compact revision %d)", tt.logPrefix(), rev)
+	plog.Printf("%s compacted storage (compact revision %d)", tt.LogPrefix(), rev)
 
-	plog.Printf("%s checking compaction (compact revision %d)", tt.logPrefix(), rev)
+	plog.Printf("%s checking compaction (compact revision %d)", tt.LogPrefix(), rev)
 	if err = tt.cluster.checkCompact(rev); err != nil {
-		plog.Warningf("%s checkCompact error (%v)", tt.logPrefix(), err)
+		plog.Warningf("%s checkCompact error (%v)", tt.LogPrefix(), err)
 		return err
 	}
 
-	plog.Printf("%s confirmed compaction (compact revision %d)", tt.logPrefix(), rev)
+	plog.Printf("%s confirmed compaction (compact revision %d)", tt.LogPrefix(), rev)
 	return nil
 }
 
 func (tt *tester) defrag() error {
-	plog.Printf("%s defragmenting...", tt.logPrefix())
+	plog.Printf("%s defragmenting...", tt.LogPrefix())
 	if err := tt.cluster.defrag(); err != nil {
-		plog.Warningf("%s defrag error (%v)", tt.logPrefix(), err)
+		plog.Warningf("%s defrag error (%v)", tt.LogPrefix(), err)
 		if cerr := tt.cleanup(); cerr != nil {
 			return fmt.Errorf("%s, %s", err, cerr)
 		}
 		return err
 	}
 
-	plog.Printf("%s defragmented...", tt.logPrefix())
+	plog.Printf("%s defragmented...", tt.LogPrefix())
 	return nil
 }
 
-func (tt *tester) logPrefix() string {
+func (tt *tester) LogPrefix() string {
```

Comment:
keep this `func (tt *tester) logPrefix() string` but add `func (tt *tester) LogPrefix() string { return tt.logPrefix() }` to keep the diff small? Exporting `LogPrefix()` doesn't seem like a good interface for the long term so it's probably better not to bake it into this file too much.